apps/teams/views.py

```diff
@@ -43,3 +43,24 @@ def team_list(request, challenge_pk):
             response_data = serializer.data
             return Response(response_data, status.HTTP_201_CREATED)
         return Response(serializer.errors, status.HTTP_400_BAD_REQUEST)
+
+
+@api_view(['GET'])
+@permission_classes((permissions.IsAuthenticated,))
+@authentication_classes((TokenAuthentication,))
+def team_detail(request, challenge_pk, pk):
+    try:
+        challenge = Challenge.objects.get(pk=challenge_pk)      # noqa
```

Comment:
No, autopep8 automatically modifies the code to follow the pep8 styling. Can you look at the 2nd point on this link https://github.com/Cloud-CV/EvalAI/blob/master/CONTRIBUTING.md